packages/themes/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@livechat/design-system-themes",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
I think that the version should be changed to `2.0.0-beta.0`
- `2.0.0` because this is a breaking change and if some one has this package installed with `^` before version it will be updated for him
- `beta` because the colors were not selected yet for `light` theme